gfx/RetroDaysTileset/tile_config.json

```diff
@@ -20857,75 +21104,75 @@
           ]
         },
         {
-          "id": "overlay_mutation_bio_armor_head",
```

Comment:
I think you overwrote some recent bionics and mutation additions here, was that intentional?  It looks like most of them are the same except for being reordered, with the exception of the claw mutations.